platform_plugin_saleor/services/helpers.py

```diff
@@ -0,0 +1,82 @@
+"""
+TO-DO
+"""
+import os
+import time
+from base64 import b64encode
+from functools import cache
+
+from django.conf import settings
+from hashids import Hashids
+
+from platform_plugin_saleor.saleor_client.client import SaleorApiClient
+
+
+def get_saleor_api_client_instance():
+    """
+    TO-DO
+    """
+    @cache
+    def generate_api_client(base_url: str, token: str):
+        """
+        TO-DO
+        """
+        return SaleorApiClient(
+            base_url=base_url,
+            token=token,
+        )
+
+    return generate_api_client(
+        base_url=settings.SALEOR_API_URL,
+        token=settings.SALEOR_API_TOKEN,
+    )
+
+
+def get_or_create_saleor_user(user) -> dict:
+    """
+    TO-DO
+    """
+    client = get_saleor_api_client_instance()
+    saleor_user = client.get_user_by_email(user.email)
+
+    if not saleor_user:
```

Comment:
I am having problem with creation user.

```suggestion
    if not saleor_user["user"]:
```
First to enter in the if block I need that.
Then
![image](https://github.com/user-attachments/assets/c8d94bcb-18f8-44bb-920e-0c261425a895)